app/models/stripe_charge.rb

```diff
@@ -40,7 +40,7 @@ def to_builder(*expand) # rubocop:disable Metrics/AbcSize
 				json.created payment.date.to_i
 
 				json.stripe_id stripe_id
-				
```

Comment:
What's going on here? If possible, let's avoid this just to simplify the changes and put any formatting issues that don't relate to stripe refund into another PR upstream.

Reply:
It was because of rubocop complaints!

Reply:
Ah, well, let's keep it then.